Makefile

```diff
@@ -128,6 +139,17 @@ dumpvar:
 	fi
 	@echo "$(VAR) is '$($(VAR))'"
 
+# eslint ftw
+ESLINT = ./node_modules/.bin/eslint
+$(ESLINT):
+	$(RUN_NPM_INSTALL)
+
+check:: check-eslint
+
+.PHONY: check-eslint
+check-eslint: $(ESLINT)
+	@./node_modules/.bin/eslint ./
```

Comment:
Is it possible to add a "npm script" in `package.json` named "eslint" and run `npm run-script eslint` here instead? That would remove the need for hardcoding paths to `./node_modules/.bin/eslint`